src/connectors/ona-api/aggregateFormData.js

```diff
@@ -154,6 +162,14 @@ function processFormData(formData, indicatorField, aggregateOptions) {
         prevSumTotal = previousPeriodGroupData[previousPeriodGroupData.length - 1][indicatorField]
           || 0;
         prevTotal = previousPeriodGroupData[previousPeriodGroupData.length - 1].total || 0;
+        if (extraProps && extraProps.length) {
+          extraProps.forEach((p, x) => {
+            prevExtraPropsSumTotal[x] = previousPeriodGroupData[
+              previousPeriodGroupData.length - 1][p]
+              || 0;
+            return prevExtraPropsSumTotal;
```

Comment:
@KipSigei - What is this returning to? I didn't know you could return something in a forEach